docs/excel/excel-add-ins-troubleshooting.md

```diff
@@ -41,6 +41,56 @@ The following APIs are affected by this workbook switch:
 
 See [Coauthoring in Excel add-ins](co-authoring-in-excel-add-ins.md) for patterns to use with events in a coauthoring environment. The article also discusses potential merge conflicts when using certain APIs, such as [`TableRowCollection.add`](/javascript/api/excel/excel.tablerowcollection#add-index--values-).
 
+## Known Issues
+
+### Binding events ID discrepancy
+
+Both [BindingDataChangedEventArgs.binding](/javascript/api/excel/excel.bindingdatachangedeventargs#binding) and [BindingSelectionChangedEventArgs.binding](/javascript/api/excel/excel.bindingselectionchangedeventargs#binding) return a temporary `Binding` object that contains the ID of the `Binding` object that raised the event. Use this ID with `BindingCollection.getItem(id)` to retrieve the `Binding` object that raised the event.
+
+The following code sample first shows how to assign an event listener to a binding, and then call the `getBindingId` method when the `onDataChanged` event is triggered. In the `getBindingId` method, the code sample then demonstrates how to use the ID of the temporary `Binding` object to retrieve the `Binding` object that raised the event.
```

Comment:
Consider leading with the reason the sample exists, before going into more detail.
```suggestion
The following code sample shows how to use this temporary binding ID to retrieve the related `Binding` object. In the sample, an event listener is assigned to a binding. The listener calls the `getBindingId` method when the `onDataChanged` event is triggered. The `getBindingId` method uses the ID of the temporary `Binding` object to retrieve the `Binding` object that raised the event.
```